src/Quoter/Quoter.cs

```diff
@@ -385,6 +385,15 @@ private ApiCall QuoteList(IEnumerable syntaxList, string name)
             return codeBlock;
         }
 
+        private static SyntaxKind GetContextualKind(SyntaxToken value)
+        {
+            var property = typeof(SyntaxToken).GetProperty("RawContextualKind",
+                BindingFlags.NonPublic | BindingFlags.Instance);
+            var result = property.GetValue(value);
+
+            return (SyntaxKind)(int)result;

```

Comment:
Can you just use GetContextualKeywordKind with the text of the token instead?

Reply:
Thanks - I updated the implementation...